### ISSUE TITLE
how often are routing numbers list updated? (question)

### ISSUE THREAD

**Issue by jasan-s** [braintree/us-bank-account-validator#17]
also curious if this is being used by brain tree in production.

**Comment by crookedneighbor** [comment]
I don't believe the routing list has been updated since the original creation of the library. This module is not used by any Braintree products in production (though we may use it when Drop-in supports bank accounts).

I'm looking in to updating the module to pull in valid routing numbers dynamically instead of hard coding them.

**Comment by jasan-s** [comment]
@crookedneighbor  I need to use routing validation in project.  And don't want to deny valid routing numbers, can you please share details on where to grab valid routing numbers dynamically until this lib is updated. Thanks in advance

**Comment by crookedneighbor** [comment]
I don't have that piece yet. I found out today that technically routing numbers can update as frequently as every day 🙃

**Comment by jasan-s** [comment]
very interesting. seems frbservices used to provide downloadable list of numbers.  I would love to stay in the loop if you make any progress

**Comment by crookedneighbor** [comment]
After talking it over with our bank payments team, we've come to the conclusion that it'd be better for us to not validate against specific routing numbers (as that list will just keep growing perpetually), so we'll likely do a major version release that drops validation for specific routing numbers.

**Comment by kickbk** [comment]
@crookedneighbor, what are your thoughts about this? https://github.com/Chris-ScoutCorps/bank-routing-number-validator/blob/master/index.js

**Comment by phlickey** [comment]
I've opened a PR implementing check digit validation.
https://github.com/braintree/us-bank-account-validator/pull/29
